src/main/java/org/frc571/bradley/Controller.java

```diff
@@ -1,9 +1,10 @@
 package org.frc571.bradley;
 
 public class Controller {
-    public static double ApplyJoystickDeadzone(double value, double deadzone) {
+    public static double ApplyJoystickDeadzone(double rawValue, double deadzone) {
         double result;
         double validRange = Constants.ControlConstants.MAX_AXIS - deadzone;
+        double value = Math.abs(rawValue);

```

Comment:
This change would require me to call Math.abs twice instead of once. As it is now, the function works and is easy-to-read. I do not suggest modifying it.